Arike/daily_dashboard/ops_tab/new_patients_registered.md

```diff
@@ -0,0 +1,167 @@
+# KPI: Count of New Patients Registered
+
+```sql 
+
+WITH patient_dates AS (
+    SELECT DISTINCT 
+        DATE(created_date) AS patient_date
+    FROM emr_patient
+    WHERE deleted = FALSE
+      [[AND {{created_date}}]]
+),
+
+
+extended_patient_dates AS (
+    SELECT patient_date
+    FROM patient_dates
+    UNION ALL
+    SELECT MAX(patient_date) + INTERVAL '1 day'
+    FROM patient_dates
+),
+
+shifted_patients AS (
+    SELECT
+        id AS patient_id,
+        created_date,
+        instance_tags,
+        CASE
+            WHEN created_date::time >= '08:00:00'
+                 AND created_date::time < '20:00:00' THEN 'day'
+            ELSE 'night'
+        END AS shift,
+        CASE
+            WHEN created_date::time >= '00:00:00'
+                 AND created_date::time < '08:00:00'
+                THEN (created_date::date - INTERVAL '1 day')
+            ELSE created_date::date
+        END AS shift_date
+    FROM emr_patient
+    WHERE deleted = FALSE
+      AND DATE(created_date) IN (SELECT patient_date FROM extended_patient_dates)
+      AND NOT (
+          DATE(created_date) = (SELECT MAX(patient_date) FROM extended_patient_dates)
+          AND created_date::time > '08:00:00'
+      )
+),
+
+patient_tags AS (
+    SELECT
+        sp.patient_id,
+        MAX(CASE WHEN et.parent_id = 55 THEN et.display END) AS zone,
+        MAX(CASE WHEN et.parent_id = 18 THEN et.display END) AS frequency_of_visit,
+        MAX(CASE WHEN et.parent_id = 26 THEN et.display END) AS cross_subsided_model
+    FROM shifted_patients sp
+    LEFT JOIN LATERAL unnest(sp.instance_tags) AS tag_id ON TRUE
+    LEFT JOIN emr_tagconfig et
+        ON et.id = tag_id
+       AND et.deleted = FALSE
+       AND et.parent_id IN (55, 18, 26)
+    GROUP BY sp.patient_id
+)
+
+SELECT COUNT(*) AS count
+FROM shifted_patients sp
+LEFT JOIN patient_tags pt ON pt.patient_id = sp.patient_id
+WHERE sp.shift_date IN (SELECT patient_date FROM extended_patient_dates)
+  [[AND (pt.zone) ILIKE ({{zone_filter}})]]
```

Comment:
let's convert the metabase filters to sql 👍 Allows others to just run the sql without being familiar with metabase.